auction-house/hooks/useListings.ts

```diff
@@ -0,0 +1,59 @@
+import {Listing,} from '@metaplex-foundation/js'
+import {useCallback, useMemo, useState} from 'react'
+import {useBoolean} from '@chakra-ui/react'
+
+import {PublicKey} from "@solana/web3.js";
```

Comment:
I think prettier didn't run here and in other files there must be space between curly braces